charts/spamoor-cronjobs/values.yaml

```diff
@@ -7,17 +7,20 @@ jobs:
       runAsGroup: 1000
       fsGroup: 2000
     image:
-      repository: curlimages/curl
-      tag: 8.8.0
+      repository: spamooor
+      tag: latest
       imagePullPolicy: Always
-    env:
-    - name: URL_VAR
-      value: "https://curl.se"
-    schedule: "0 * * * *"
+    schedule: "*/10 * * * *" # run every 10mins
     args:
-      - "-L"
-      - "-v"
-      - "URL_VAR"
+      - "spamooor"
```

Comment:
I don't think `spamoor` is necessary here. But we'll probably need to add the yaml above to be something like: 

```
command: ["/app/spamoor"]
args: 
      -
      -
      ...
```